src/Test/Perf/util/test_util.csx

```diff
@@ -200,6 +216,41 @@ ProcessResult ShellOut(
     };
 }
 
+ProcessResult ShellOutUsingShellExecute(
```

Comment:
@KevinH-MS  I will extract `CopyDirectory` to a helper and make necessary modification to `ShellOut` to make it work with it. The interesting part is, how is `CopyDirectory` in `fetch_build.csx` passing because 0-7 exit code in Robocopy means success and anything above 7 is error. `ShellOut` always assumes that the success is when the `exitcode` is `0`, which in Robocopy means `No errors occurred, and no copying was done. The folders were in sync`.